flake.nix

```diff
@@ -259,7 +260,7 @@
           # can't give Nix those hashes and must instead determine it ourselves.
           # this means that we will have to update this SHA if the clickhouse
           # version changes.
-          sha256 = "1lgxwh67apgl386ilpg0iy5xkyz12q4lgnz08zswjbxv88ra0qxj";
+          sha256 = "0wx8w9sdms5hsc9f835ivsissf15wjzdb9cvxr65xdi384i9pkzx";
```

Comment:
lowkey would love to switch these from md5 to sha2 instead, so we can drop our md5 dependency. But I realize that's not just a "nix flake" problem, that's like, a "fix how we're building our third party software" problem

Reply:
I would also really like that --- then, the Nix flake would require substantially less manual intervention when we update Clickhouse. For the deps that we have SHA2 hashes for, the Nix stuff should "just work" when the hash files change (modulo cases where other library deps appear, like `mgd` adding an `openssl-dev` dep).